Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -150,6 +150,21 @@ inline void ClearStrokeStyle() {
         return GStateStack().top();
     }
 
+    inline bool HasPath() {
+        return _impl.currentPath != nullptr;
+    }
+
+    inline CGMutablePathRef Path() {
+        if (!_impl.currentPath) {
+            _impl.currentPath.reset(CGPathCreateMutable());
+        }
+        return _impl.currentPath.get();
+    }
+
+    inline void SetPath(CGMutablePathRef path) {
```

Comment:
>   inline void SetPath(CGMutablePathRef path) { [](start = 1, length = 47)

could we just switch over to the standard of having getters and setters to keep encapsulation throughout the code?
We have a mix of direct access and using getters and setters. It would be good to just follow one standard (encapsulation works the best here).